snapcraft/_store.py

```diff
@@ -605,7 +605,7 @@ def _push_delta(snap_name, snap_filename, source_snap):
     }
 
     try:
-        logger.info("Pushing delta {}.".format(delta_filename))
+        logger.debug("Pushing delta {}.".format(delta_filename))
```

Comment:
add !r

Reply:
I tried both variants (with/without quotes) and found the quoteless version to be aesthetically more pleasant, but that's highly subjective. As the project maintainer it's your call to decide, so I'll put them back.